tests/metrics/test_bleurt20.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -eu #o pipefail
```

Comment:
Please add description and tags as in other tests. If this is CPU-only test, please make sure to add relevant tag - "cpu" if I'm not wrong.